skycatalogs/skyCatalogs.py

```diff
@@ -653,6 +722,9 @@ def open_catalog(config_file, mp=False, skycatalog_root=None, verbose=False):
     '''
     # Get LSST bandpasses in case we need to compute fluxes
     _ = load_lsst_bandpasses()
+    base_dir = os.path.dirname(config_file)
+    YamlIncludeConstructor.add_to_loader_class(loader_class=yaml.SafeLoader,
```

Comment:
Ah.  It's allowing things like this (from `skycatalogs/data/ci_sample/skyCatalog_top.yaml`)  in the yaml configuration:
`object_types:
  bulge: !include cosmodc2_bulge.yaml
  disk: !include cosmodc2_disk.yaml
  galaxy: !include cosmodc2_galaxy.yaml
  knots: !include cosmodc2_knots.yaml
  star: !include dc2_star.yaml
`

Reply:
In this example the included yaml files are in the same directory as the top one, but they don't have to be.
Ideally these includes are generated by whatever program generates the data.  Then the yam for that data and the data can be in, e.g., a subdirectory of the top-level directory.  The subdirectory doesn't need to be a physical subdirectory; it can be a sym link.